sktime/transformations/base.py

```diff
@@ -110,6 +110,7 @@ class BaseTransformer(BaseEstimator):
         "scitype:instancewise": True,  # is this an instance-wise transform?
         "capability:inverse_transform": False,  # can the transformer inverse transform?
         "capability:inverse_transform:range": None,
+        "capability:inverse_transform:exact": True,
```

Comment:
Question: is `True` a good default? I thought most transformers do not even override `_inverse_transform`, and in some cases inverse actually does not exist.

(assuming `True` means this and it's reverse hold good: `FittedTransformerInstance.inverse_transform(self.transform(X)) = X`)

Reply:
> Question: is `True` a good default? 

It is the current default and the setting that will not change anything for downstream tests.

> I thought most transformers do not even override `_inverse_transform`, and in some cases inverse actually does not exist (example: holiday ones).

The tag does something only when `inverse_transform` exists - namely, turn off the test for it being an exact inverse, and it does nothing beyond that.

> (assuming `True` means this and it's reverse hold good: `FittedTransformerInstance.inverse_transform(self.transform(X)) = X`)

More precisely, it checks that `inverse_transform` is a left inverse to `transform` under function composition once the estimator has been fitted.